docs/_includes/events.html

```diff
@@ -0,0 +1,24 @@
+<!-- Events -->
+<section class="page-section" id="events">
+    <div class="container text-center">
+      <h2 class="page-section-heading text-uppercase">Hacktoberfest & GrimoireLab</h2>
+      <div class="row text-left mt-5">
+        <div class="col-lg-5 ml-auto">
+          <p>Help driving growth of open source and make positive contributions to the community! This is how participants from all skill levels can help: </p>
+          <ol>
+            <li class="mb-3"><b>Visit <a href="https://hacktoberfest.digitalocean.com/">hacktoberfest website</a> and sign in with your github account:</b> Once linked, hacktoberfest will send you an email confirmation that you have officially been registered.</li>
+            <li class="mb-3"><b>Choose your favourite <a href="https://github.com/chaoss/grimoirelab/issues">Grimoirelab issue</a> and submitt your Pull Request:</b> issues will be tagged using <button class="button button4">Hacktoberfest</button> label.</li>
+            <li class="mb-3"><b>Wait for your PR to be merged:</b> You will need a minimum of 4 Pull Requests merged to fully complete the hacktoberfest challenge.</li>
+          </ol>
+        </div>
+        <div class="col-lg-7 ml-auto">
+          <img src="img/hacktoberfest.svg" alt="hacktoberfest" class="img-fluid">
+        </div>
+      </div>
+      <div class="text-center mt-4">
+        <a class="btn btn-xl btn-outline-primary" href="https://github.com/chaoss/grimoirelab/blob/master/CONTRIBUTING.md">
+          <svg class="bi bi-terminal-fill h2" width="1em" height="1em" viewBox="0 0 16 16" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+            <path fill-rule="evenodd" d="M0 3a2 2 0 012-2h12a2 2 0 012 2v10a2 2 0 01-2 2H2a2 2 0 01-2-2V3zm9.5 5.5h-3a.5.5 0 000 1h3a.5.5 0 000-1zm-6.354-.354L4.793 6.5 3.146 4.854a.5.5 0 11.708-.708l2 2a.5.5 0 010 .708l-2 2a.5.5 0 01-.708-.708z" clip-rule="evenodd"/>
+          </svg>
+           Contribute to Open Source</a>
+    </div>
```

Comment:
I think there are several tags not closed. At least, I miss:
```
</div>
</section>
```

Reply:
sorry for that! its seems is the only one missing :+1: